packages/core/src/features/extensions/enabled/common/storage.injectable.ts

```diff
@@ -24,13 +31,25 @@ const enabledExtensionsPersistentStorageInjectable = getInjectable({
 
     return createPersistentStorage<EnabledExtensionsStorageModal>({
       configName: "lens-extensions",
-      fromStore: action(({ extensions = [] }) => {
+      fromStore: action(({ extensions: rawExtensions = [] }) => {
+        const extensions = rawExtensions
+          .map(([key, value]) => {
+            const verification = stateModel.safeParse(value);
+
+            if (!verification.success) {
+              return undefined;
+            }
+
+            return [key, verification.data] as const;
+          })
+          .filter(isDefined);
+
         state.replace(extensions);
       }),
       toJSON: () => ({
         extensions: [...toJS(state)],
       }),
-      projectVersion: di.inject(storageMigrationVersionInjectable, enabledExtensionsMigrationDeclarationInjectionToken),
+      projectVersion: di.inject(enabledExtensionsPersistentStorageVersionInitializable.stateToken),
```

Comment:
Why is the field called `projectVersion`? It might be clearer if it were `storeVersion` or similar. Without digging deeper in the code one might think this could be the package version.

Reply:
That is because the underlying library means for it to be the actual version of the project (which is the default value too).

We could change it internally, but I don't know if it is worth it.